include/swift/AST/DiagnosticsFrontend.def

```diff
@@ -573,7 +573,7 @@ ERROR(dont_enable_interop_and_compat,none,
       "'-cxx-interoperability-mode'; remove '-enable-experimental-cxx-interop'", ())
 
 NOTE(valid_cxx_interop_modes,none,
-      "valid arguments to '-cxx-interoperability-mode=' are %0", (StringRef))
+      "valid arguments to '%0' are %1", (StringRef, StringRef))
```

Comment:
Nit: If we only have 2 options here, maybe a `%select{` would be a better way to take the arguments.

Reply:
My argument in favor of taking a `StringRef` here is that, at the call site, `arg->getSpelling()` means something at face-value, rather than `true`/`false` or `1`/`0`. But I don't feel very strongly about this. What do you think?

Reply:
Orthogonally I'm even wondering whether we should combine this with this other note:

```
NOTE(note_valid_swift_versions, none,
     "valid arguments to '-swift-version' are %0", (StringRef))
```

In fact, `valid arguments to <flag> are <options>` seems like a _very_ re-usable note and it would be nice to de-duplicate some of that.

But I think that can also be done in a follow-up patch.

Reply:
Good point. I am fine with this as is then.